### ISSUE TITLE
Java code snippets are not build-flagged

### ISSUE THREAD

**Issue by ig-ealbert** [Infragistics/convertwire#38]
Java code snippets are not build-flagged.

datachart-category-line-series.ja-JP.adoc

This might be related to #25, although I think it's different than what I had originally reported.

**Comment by ig-ealbert** [comment]
datachart-category-spline-series.ja-JP.adoc

**Comment by craigshoemaker** [comment]
The current converter is wrapping the code in build flags, but they are flagged as Android. For example the code listing for `datachart-category-line-series.ja-JP.adoc` has the following listing:

```
ifdef::android[]
*In Java:*

[source,js]
----
EnergyProductionDataSample data = new EnergyProductionDataSample();
DataChartView chart = new DataChartView(rootView.getContext());
NumericYAxis yAxis = new NumericYAxis();
CategoryXAxis xAxis = new CategoryXAxis();
xAxis.setDataSource(data);
xAxis.setLabel("Country");

LineSeries series = new LineSeries();
series.setDataSource(data);
series.setValueMemberPath("Coal");
series.setTitle("Coal");
series.setXAxis(xAxis);
series.setYAxis(yAxis);

chart.addAxis(xAxis);
chart.addAxis(yAxis);
chart.addSeries(series);
rootView.addView(chart);
----

endif::android[]
```

Were you expecting it to be build flagged as 'Java' instead? 

I verified that both the files you noted here have listings flagged at Android and they are correctly flagged in AsciiDoc in the latest build - so let me know if you need it to do anything else.

**Comment by ig-ealbert** [comment]
It's not build flagged at all in the latest version you sent out.

```
endif::winforms[]

*Java の場合:*

[source,js]
----
EnergyProductionDataSample data = new EnergyProductionDataSample();
DataChartView chart = new DataChartView(rootView.getContext());
NumericYAxis yAxis = new NumericYAxis();
CategoryXAxis xAxis = new CategoryXAxis();
xAxis.setDataSource(data);
xAxis.setLabel("Country");

LineSeries series = new LineSeries();
series.setDataSource(data);
series.setValueMemberPath("Coal");
series.setTitle("Coal");
series.setXAxis(xAxis);
series.setYAxis(yAxis);

chart.addAxis(xAxis);
chart.addAxis(yAxis);
chart.addSeries(series);
rootView.addView(chart);
```

**Comment by ig-ealbert** [comment]
Looks like it's build-flagged only in the English topics.  Is there something you're searching for that is hardcoded in English?

**Comment by craigshoemaker** [comment]
There was a small variation between the English and Japanese converters. To remedy this I created a new set of rules for cleaning up HTML and now it works on all topics regardless of the language. 

Closed by 7345c182069d8121817b0c9a4a2489ab5b8dc856
